doc/ansible/dev/finalizers.md

```diff
@@ -56,8 +56,8 @@ or `role` was set for the finalizer.
 Here are a few examples of `watches.yaml` files that specify a finalizer:
 
 ### Run top-level playbook or role with new variables
+
 ```yaml
----
```

Comment:
Nit: The `watches.yaml` file scaffolded by the ansible operator contains the `---` YAML document start marker, so we should leave this in to be consistent with the scaffolded file.